workmail-message-flow-state-machine/README.md

```diff
@@ -4,43 +4,34 @@ This application enables you to use AWS Step Functions to orchestrate multiple A
 
 ## Setup
 1. Deploy this application via [AWS Serverless Application Repository](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:489970191081:applications~workmail-message-flow-state-machine).
+    1. [Optional] Enter `MachineStateForOutput` to specify state of the Step Function whose output should be returned back to WorkMail. By default, output of the succeeded execution event is used.
+    2. [Optional] Enter `WaitTimeForExecution` to specify number of seconds to wait for result after the state machine is first invoked, useful when step function finishes within few seconds. By default, value is 1 second.
```

Comment:
... to specify **the** number of seconds to wait for **the** result after...

... useful when **the Step Function** finishes...

... By default, **the** value is **one** second.